src/pybind/mgr/cephadm/module.py

```diff
@@ -2275,11 +2286,13 @@ def _ok_to_stop(remove_daemon_hosts: Set[orchestrator.DaemonDescription]) -> boo
             # let's find a subset that is ok-to-stop
             remove_daemon_hosts.pop()
         for d in remove_daemon_hosts:
+            r = True
```

Comment:
I'm confused, why set this before `_remove_daemon`?

Reply:
no great reason.  It was just a result of reworking that code and focusing on the return value paths.  It would be at least as good and arguably better after.